include/proxy-wasm/context.h

```diff
@@ -229,6 +229,7 @@ class ContextBase : public RootInterface,
   WasmResult log(uint32_t /* level */, string_view /* message */) override {
     return unimplemented();
   }
+  uint32_t getLogLevel() override { return 0; }
```

Comment:
probably want to default this to WARNING or INFO (2) rather than trace.

Reply:
Good idea. Ack!